indico/htdocs/js/indico/modules/designer.js

```diff
@@ -470,20 +471,33 @@
             },
             clear_background: !template.background_url,
             title: $('.js-template-name').val(),
+            disassociate_backsides: showConfirmationDialog,
             backside_template_id: backsideTemplateID
         };
 
-        $.ajax({
-            url: location.pathname,
-            data: JSON.stringify(templateData),
-            contentType: 'application/json',
-            method: 'POST',
-            complete: IndicoUI.Dialogs.Util.progress(),
-            error: handleAjaxError,
-            success: function(data) {
-                handleFlashes(data, true);
-            }
-        });
+        if (showConfirmationDialog) {
+            confirmPrompt(
+                $T("Are you sure you want to change the dimensions of this template? " +
```

Comment:
` $T.gettext()` and you most likely cannot use `+` inside translated strings without breaking the extraction. However, having these two sentences in separate gettext calls doesn't seem too bad.